asn1/InterledgerProtocol.asn

```diff
@@ -10,12 +10,9 @@ IMPORTS
     Address,
     Timestamp
     FROM InterledgerTypes
-
-    MetadataEntry
-    FROM MetadataEntry
 ;
 
-InterledgerProtocolPayment ::= SEQUENCE {
+InterledgerProtocolPaymentMessage ::= SEQUENCE {
```

Comment:
I reverted back to `InterledgerProtocolPayment` in `InterledgerProtocol.asn` and changed `GenericPacket.asn` to match